Dfe.Academies.Academisation.Data/ProjectAggregate/ProjectListGetDataQuery.cs

```diff
@@ -42,32 +42,36 @@ private static IQueryable<ProjectState> FilterByStatus(string[]? states, IQuerya
 
 		private static IQueryable<ProjectState> FilterByUrn(int? urn, IQueryable<ProjectState> queryable)
 		{
-			if (urn.HasValue) queryable = queryable.Where(p => p.Urn == urn);			
+			if (urn.HasValue) queryable = queryable.Where(p => p.Urn == urn);
 
 			return queryable;
 		}
 
 		private static IQueryable<ProjectState> FilterBySchool(string? title, IQueryable<ProjectState> queryable)
 		{
 			if (!string.IsNullOrWhiteSpace(title)) queryable = queryable.Where(p => p.SchoolName!.ToLower().Contains(title!.ToLower()));
-			
+
 			return queryable;
 		}
 
-		private static IQueryable<ProjectState> FilterByDeliveryOfficer(string[]? deliveryOfficers, IQueryable<ProjectState> queryable,
-			AcademisationContext context)
+		private static IQueryable<ProjectState> FilterByDeliveryOfficer(string[]? deliveryOfficers, IQueryable<ProjectState> queryable)
 		{
 			if (deliveryOfficers != null && deliveryOfficers.Any())
 			{
 				var lowerCaseDeliveryOfficers = deliveryOfficers.Select(officer => officer.ToLower());
-				queryable = queryable
-					.Where(p =>
-						!string.IsNullOrEmpty(p.AssignedUserFullName) && lowerCaseDeliveryOfficers.Contains(p.AssignedUserFullName.ToLower()));
 
 				if (lowerCaseDeliveryOfficers.Contains("not assigned"))
 				{
-					var unAssignedProjects = context.Projects.Where(p => string.IsNullOrEmpty(p.AssignedUserFullName));
-					queryable = queryable.Concat(unAssignedProjects);
+					// Query by unassigned or assigned delivery officer
```

Comment:
@dneed-nimble  I want to make sure that you've reviewed this bit.
Do you think it solves the issue we talked about yday evening?

Reply:
I do, yes.